addons/website_slides/wizard/slide_channel_invite.py

```diff
@@ -19,34 +19,50 @@ class SlideChannelInvite(models.TransientModel):
 
     # composer content
     attachment_ids = fields.Many2many('ir.attachment', string='Attachments')
+    send_email = fields.Boolean('Send Email')
     # recipients
     partner_ids = fields.Many2many('res.partner', string='Recipients')
     # slide channel
-    channel_id = fields.Many2one('slide.channel', string='Slide channel', required=True)
+    channel_id = fields.Many2one('slide.channel', string='Course', required=True)
+    channel_invite_url = fields.Char('Course Link', compute='_compute_channel_invite_url')
+    channel_visibility = fields.Selection(related='channel_id.visibility')
+    channel_published = fields.Boolean(related='channel_id.is_published')
+    # membership
+    with_enrollment = fields.Boolean(
```

Comment:
I see it is used a lot, so well... not a fan of the "with_enrollment" (enroll_mode ? invite_only ?), but maybe you don't want to rewrite half of your PR :disguised_face: